### ISSUE TITLE
Need to clean up OMPD CMake files for OMPD

### ISSUE THREAD

**Issue by mcfadden8** [OpenMPToolsInterface/LLVM-openmp#47]
The current ompd CMake files produce the following warnings that need to be cleaned up before they may be merged back upstream:

``` bash
OMPD_SYMBOLS_INC_DIRS == /g/g0/martymcf/.session.ray/build/linux-rhel7-ppc64le/llvm.ompd/include;/g/g0/martymcf/.session.ray/src/llvm.ompd/include;/g/g0/martymcf/.session.ray/build/linux-rhel7-ppc64le/llvm.ompd/projects/openmp/runtime/src;/g/g0/martymcf/.session.ray/src/llvm.ompd/projects/openmp/runtime/src;/g/g0/martymcf/.session.ray/src/llvm.ompd/projects/openmp/runtime/src/i18n;/g/g0/martymcf/.session.ray/src/llvm.ompd/projects/openmp/runtime/src/include/50;/g/g0/martymcf/.session.ray/src/llvm.ompd/projects/openmp/runtime/src/thirdparty/ittnotify
-- Looking for sqrt in m
-- Looking for sqrt in m - found
-- Looking for __atomic_load_1
-- Looking for __atomic_load_1 - not found
-- Looking for __atomic_load_1 in atomic
-- Looking for __atomic_load_1 in atomic - found
CMake Warning (dev) at projects/openmp/libompd/src/CMakeLists.txt:1 (project):
  Policy CMP0048 is not set: project() command manages VERSION variables.
  Run "cmake --help-policy CMP0048" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  The following variable(s) would be set to empty:

    PROJECT_VERSION
    PROJECT_VERSION_MAJOR
    PROJECT_VERSION_MINOR
    PROJECT_VERSION_PATCH
This warning is for project developers.  Use -Wno-dev to suppress it.
```

**Comment by jprotze** [comment]
Cannot reproduce this
